erik/calculator/jsFolder.js/mainjs.js

```diff
@@ -0,0 +1,60 @@
+var output = document.querySelector("#output_value");
+var operation = document.querySelector(".btn-operator");
+var stash;
+var numClear = true;
+let total;
+
+document.querySelectorAll(".btn_numbers").forEach(function(num) {
+  num.addEventListener("click", function(e) {
+    if (operation.value && numClear) {
```

Comment:
`numClear` is slightly ambiguous. I'd opt for something like `readyToClear` perhaps? Something that signifies a bit more clearly what's needed to happen. When I look (subjectively) at this I think: is this the number clear element, is it saying the number is cleared?...